packages/gatsby-remark-copy-linked-files/README.md

```diff
@@ -16,96 +16,93 @@ Hey everyone, I just made a sweet PDF with lots of interesting stuff in it.
 
 **When you build your site:**
 
-The `my-awesome-pdf.pdf` file will be copied to the root directory (i.e., `public/some-really-long-contenthash/my-awesome-pdf.pdf`) and the generated HTML page will be modified to point to it.
+The `my-awesome-pdf.pdf` file will be copied to the `public` folder (i.e., `public/some-really-long-contenthash/my-awesome-pdf.pdf`) and the generated HTML page will be modified to point to it.
 
 > **Note**: The `my-awesome-pdf.pdf` file should be in the same directory as the markdown file.
 
----
-
-## Install plugin
+## Installation
 
-`npm install gatsby-remark-copy-linked-files`
+```shell
+npm install gatsby-remark-copy-linked-files
+```
 
-## Add plugin to Gatsby Config
+## Configuration
 
-**Default settings:**
+### Default settings
 
 Add `gatsby-remark-copy-linked-files` plugin as a plugin to [`gatsby-transformer-remark`](https://www.gatsbyjs.com/plugins/gatsby-transformer-remark/):
 
-```javascript
-// In your gatsby-config.js
-
-// add plugin by name only
-plugins: [
-  {
-    resolve: `gatsby-transformer-remark`,
-    options: {
-      plugins: [`gatsby-remark-copy-linked-files`],
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [`gatsby-remark-copy-linked-files`],
+      },
     },
-  },
-]
+  ],
+}
 ```
 
-**Custom settings:**
-
-```js
-// In your gatsby-config.js
-
-// add plugin by name and options
-plugins: [
-  {
-    resolve: `gatsby-transformer-remark`,
-    options: {
-      plugins: [
-        {
-          resolve: `gatsby-remark-copy-linked-files`,
-          options: {
-            destinationDir: `path/to/dir`,
-            ignoreFileExtensions: [`png`, `jpg`, `jpeg`, `bmp`, `tiff`],
+### Custom settings
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [
+          {
+            resolve: `gatsby-remark-copy-linked-files`,
+            options: {
+              destinationDir: `path/to/dir`,
+              ignoreFileExtensions: [`png`, `jpg`, `jpeg`, `bmp`, `tiff`],
+            },
           },
-        },
-      ],
+        ],
+      },
     },
-  },
-]
+  ],
+}
 ```
 
----
-
-## Custom set where to copy the files using `destinationDir`
+## Option: `destinationDir`
 
-By default, all files will be copied to the root directory (i.e., `public` folder) in the following format: `contentHash/fileName.ext`.
+By default, all files will be copied to the root of the `public` folder in the following format: `contentHash/fileName.ext`.
```

Comment:
Since you got rid of the other references to "root" or "root directory", maybe we should get rid of it here too?

```suggestion
By default, all files will be copied to the `public` folder in the following format: `contentHash/fileName.ext`.
```

Reply:
No, this was on purpose. It's not _somewhere_  in the public folder but at the root of the folder